torch/_export/non_strict_utils.py

```diff
@@ -94,8 +93,14 @@ def fakify(
     if not isinstance(t, torch.Tensor):
         raise ValueError(f"Unsupported input type {type(t)}")
     n_dims = len(t.shape)
+    dynamic_sizes = [
+        DimDynamic.DYNAMIC
+        if i in getattr(t, "_dynamo_weak_dynamic_indices", {})
+        else DimDynamic.STATIC
+        for i in range(n_dims)
+    ]
     symbolic_context = StatelessSymbolicContext(
-        dynamic_sizes=[DimDynamic.DYNAMIC] * n_dims,
+        dynamic_sizes=dynamic_sizes,
```

Comment:
OK, trying to understand...so you start off with mostly STATIC here.

Reply:
Yeah, for now everything here can be static except those marked with Dim.AUTO. This doesn't interfere with the old Dim() specs.